src/public/modules/forms/base/components/verifiable-field.client.component.js

```diff
@@ -81,10 +82,11 @@ function verifiableFieldController(Verification, $timeout, $interval) {
         return onVerificationFailure()
       }
 
-      await Verification.verifyOtp(
-        { transactionId: vm.transactionId },
-        { fieldId: vm.field._id, otp: otp },
-      )
+      await verifyOtp({
```

Comment:
do you not need to wrap this in `$q.when`?

Reply:
no rendering depends on this function so didn't use it

Reply:
`vm.field.signature` and `vm.display` no meh?
```
const onVerificationSuccess = (signature) => {
    lastRequested.verified = true
    lastRequested.signature = signature
    vm.field.signature = signature
    updateInputValidity(lastRequested.verified)
    updateView(STATES.VFN_SUCCESS)
  }
```

Reply:
oh ya didn't remember the `.then`. Actually it still works... is `$q` not required after all 🤔